appcircle_android_build_ui_test/1.0.2/component.yaml

```diff
@@ -0,0 +1,44 @@
+platform: Android
+buildPlatform: ReactNative, JavaKotlin, Flutter
+displayName: "Android Build for UI Testing"
+description: "Build your test applications with gradlew(./gradlew assembleDebug assembleAndroidTest)."
+docUrl: "https://docs.appcircle.io/workflows/android-specific-workflow-steps#android-build-for-ui-testing"
+requiredComponents: "appcircle_git_clone"
+precedingComponents: "appcircle_activate_ssh_key,appcircle_node_install,appcircle_npm_yarn,appcircle_android_build_version_increment"
+followingComponents: "appcircle_android_sign,appcircle_android_resign,appcircle_android_test_report,appcircle_android_post_process,appcircle_export_build_artifacts,appcircle_android_wait_emulator,appcircle_badge,appcircle_bundletool,appcircle_datatheorem,appcircle_android_appdome,appcircle_repeato,appcircle_testinium,appcircle_browserstack_espresso,appcircle_android_firebase_test_lab"
```

Comment:
If this component generates an apk or aab file that users are able to use to distribute. We can say that badge component may follow, but if it is not, the badge component not necessary.

Reply:
@boztopuz Yes, the app produces it as output. I thought it might be use to be distributed to testers.